lib/modules/manager/cnb-project-descriptor/index.ts

```diff
@@ -0,0 +1,12 @@
+import type { Category } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  commitMessageTopic: 'buildpacks project descriptor {{depName}}',
+  fileMatch: ['(^|/)project\\.toml$'],
+  pinDigests: false,
+};
+
+export const categories: Category[] = ['docker', 'buildpacks'];
+export const supportedDatasources = [DockerDatasource.id];
```

Comment:
Are there other dependency types which need later implementation?

Reply:
None at the moment. The only other potential source for buildpacks would require brand new datasource (buildpacks registry at https://registry.buildpacks.io/)